dashboards/mysql/metadata.yaml

```diff
@@ -9,3 +9,13 @@ sample_dashboards:
     related_integrations:
       - id: mysql
         platform: GCE
+  -
+    category: MySQL
+    id: prometheus
+    display_name: MySQL Prometheus Overview
+    description: |-
+      This dashboard has charts for viewing MySQL when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mysql#monitored-metrics), including Buffer Pool Operations, Buffer Pool Pages, Buffer Pool Size, Handlers, Double Writes, Page Operations, Operations, Log Operations, Row Operations, Row Locks, Locks, Commands, Sorts, and Threads from MySQL as well as charts of infrastructure related metrics for the running MySQL VMs: CPU % Top 5 VMs and MySQL VMs by Region for a count of VMs over time.
+      There is also a card with links to docs and MySQL log in Cloud Logging.
```

Comment:
This description looks copied from the existing dashboard, please update this to reflect the correct dashboard.